src/components/LearningDaysList/LearningDaysList.js

```diff
@@ -13,11 +13,11 @@ const LearningDaysList = ({ setLoading, setAlert, topics, isTeamCalendar }) => {
 
     const emptyLearningDay = {
         date: new Date().toISOString(),
-        id: null,
-        title: null,
-        firstName: null,
-        lastName: null,
-        userId: null,
+        id: NaN,
```

Comment:
I don't think so that NaN is a good value by default, why are you not using null anymore?